charts/rating/values.yaml

```
@@ -0,0 +1,80 @@
+# Default values for rating.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: "jenkinsciinfra/rating"
+  tag: "0.0.7"
+  pullPolicy: IfNotPresent
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name:
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
+  #   drop:
+  #   - ALL
+  # readOnlyRootFilesystem: true
+  # runAsNonRoot: true
+  # runAsUser: 1000
+
+service:
+  type: ClusterIP
+  port: 80
+
+# Please define ingress settings into environment variable
+ingress:
+  enabled: true
```

```suggestion
  enabled: false
```
I think it's better to disable the exposition on the internet by default, and let people enabling it when they decide to.

---

I don't know how you overload your configuration so it's up to you